scripts/test/console_output_test.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
```

Comment:
I'd just call this the "console_test.py" or "hyriseConsole_test.py" - every "test" tests "output", that info is redundant or even wrong when you consider that you also supply "input" to the console and check whether that works.